features/taxonomy/managing_taxons/reordering_taxon.feature

```diff
@@ -0,0 +1,18 @@
+@managing_taxons
+Feature: Reordering taxons
+    In order to see all ordered taxons in the store
+    As an Administrator
+    I want to browse ordered taxons
+
+    Background:
+        Given the store classifies its products as "T-Shirts" and "Watches"
+        And the store classifies its products as "Belts" and "Wallets"
+        And I am logged in as an administrator
+
+    @ui @javascript
+    Scenario: Changing order of the taxon
```

Comment:
``Changing the order of taxons`` but... where do we change that? Where is this ordering used?